__tests__/unit/collection-test.js

```diff
@@ -1,47 +1,46 @@
-import { Collection } from '@miragejs/server';
-import {module, test} from 'qunit';
+import { Collection } from '../../lib';
 
-module('Unit | Collection', function() {
-  test('it can be instantiated', function(assert) {
+describe('Unit | Collection', function() {
+  test('it can be instantiated', () => {
     let collection = new Collection('plant');
 
-    assert.ok(collection);
+    expect(collection).toBeTruthy();
   });
 
-  test('it cannot be instantiated without a modelName', function(assert) {
-    assert.throws(() => {
+  test('it cannot be instantiated without a modelName', () => {
+    expect(() => {
       new Collection();
-    }, /must pass a `modelName`/);
```

Comment:
looks like there's a `toThrowError(/must pass a `modelName`/)` we can use here. Would be good to preserve these error message checks.